org.pitivi.Pitivi.json

```diff
@@ -0,0 +1,462 @@
+{
+    "app-id": "org.pitivi.Pitivi",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.24",
+    "command": "pitivi",
+    "finish-args": ["--socket=x11",
+                    "--socket=pulseaudio",
+                    "--socket=wayland",
+                    "--socket=session-bus",
```

Comment:
No reason to pass through entire bus.

Reply:
https://github.com/flathub/flathub/pull/52#discussion_r140670984